tools/pkglint/lib/rules.ts

```diff
@@ -487,6 +487,26 @@ export class JSIIProjectReferences extends ValidationRule {
   }
 }
 
+export class NoPeerDependenciesMonocdk extends ValidationRule {
+  public readonly name = 'monocdk/no-peer';
+  public allowedPeer = ['constructs'];
+
+  public validate(pkg: PackageJson): void {
+    if (pkg.packageName !== 'monocdk') {
```

Comment:
what about aws-cdk-lib?

Reply:
aws-cdk-lib is a private package which is under active development.

Reply:
We should include aws-cdk-lib to this and update the error message.